GettingStarted/ibmdbpy_GettingStarted_1-basics.ipynb

```diff
@@ -0,0 +1,2489 @@
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>sepal_length</th>\n",
+       "      <th>sepal_width</th>\n",
+       "      <th>petal_length</th>\n",
+       "      <th>petal_width</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>sepal_length</th>\n",
+       "      <td>0.685694</td>\n",
+       "      <td>-0.042434</td>\n",
+       "      <td>1.274315</td>\n",
+       "      <td>0.516271</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>sepal_width</th>\n",
+       "      <td>-0.042434</td>\n",
+       "      <td>0.189979</td>\n",
+       "      <td>-0.329656</td>\n",
+       "      <td>-0.121639</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>petal_length</th>\n",
+       "      <td>1.274315</td>\n",
+       "      <td>-0.329656</td>\n",
+       "      <td>3.116278</td>\n",
+       "      <td>1.295609</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>petal_width</th>\n",
+       "      <td>0.516271</td>\n",
+       "      <td>-0.121639</td>\n",
+       "      <td>1.295609</td>\n",
+       "      <td>0.581006</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "              sepal_length  sepal_width  petal_length  petal_width\n",
+       "sepal_length      0.685694    -0.042434      1.274315     0.516271\n",
+       "sepal_width      -0.042434     0.189979     -0.329656    -0.121639\n",
+       "petal_length      1.274315    -0.329656      3.116278     1.295609\n",
+       "petal_width       0.516271    -0.121639      1.295609     0.581006"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Compute the covariance matrix\n",
+    "idadf.cov()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Data selection and projection__\n",
+    "\n",
+    "When working with an IdaDataFrame, you can use classic slicing like with NumPy arrays and Pandas DataFrames."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Note: Use of an indexer*\n",
+    "\n",
+    "A warning would be raised by this line of code.\n",
+    "\n",
+    "> idadf_new1 = idadf[0:9] \n",
+    "\n",
+    " We want to select the first ten rows but \"Row order is not guaranteed if no indexer was given and the dataset was not sorted\".\n",
+    "Of course, using row indices only makes sense if an ID column is provided. Otherwise, the selection is non-deterministic. Here an index has not been provided so far, that is why a warning would be shown.\n",
```

Comment:
If you try to use idadf[0:9] without having set an indexer before it will return an error. 
So ok perhaps not "of course" but ibmdbpy requires an indexer column to have been explicitly assigned.

Reply:
pandas has no probel with it but this is not the case of ibmdbpy

Reply:
> If you try to use idadf[0:9] without having set an indexer before it will return an error.

That means that "a warning would be shown" is not correct.

>  So ok perhaps not "of course" but ibmdbpy requires an indexer column to have been explicitly assigned.

In that case, why not just write "ibmdbpy requires an indexer column to have been assigned. For our use case, `add_column_id` is good enough, so we will use that now."?

Reply:
I have changed it to: 

> For ibmdbpy, using row indices only makes sense if an ID column is provided.